src/coreclr/vm/interpexec.cpp

```diff
@@ -47,265 +51,269 @@ void InterpExecMethod(InterpreterFrame *pInterpreterFrame, InterpMethodContextFr
     const int32_t *targetIp;
 
 MAIN_LOOP:
-    while (true)
+    try
     {
-        // Interpreter-TODO: This is only needed to enable SOS see the exact location in the interpreted method.
-        // Neither the GC nor the managed debugger needs that as they walk the stack when the runtime is suspended
-        // and we can save the IP to the frame at the suspension time.
-        // It will be useful for testing e.g. the debug info at various locations in the current method, so let's
-        // keep it for such purposes until we don't need it anymore.
-        pFrame->ip = (int32_t*)ip;
-
-        switch (*ip)
+        INSTALL_MANAGED_EXCEPTION_DISPATCHER;
+        INSTALL_UNWIND_AND_CONTINUE_HANDLER;
```

Comment:
Is `INSTALL_UNWIND_AND_CONTINUE_HANDLER` going to break exception filter semantics when we have mix of interpreted and JIIT/AOT frames on the stack?

Reply:
I guess it works fine with filter. I do not understand why we need `INSTALL_UNWIND_AND_CONTINUE_HANDLER` here. What is going to break if `INSTALL_UNWIND_AND_CONTINUE_HANDLER` is deleted here?

Reply:
That's for handling exceptions coming out of calls to native runtime. The INSTALL_MANAGED_EXCEPTION_DISPATCHER does nothing for Windows. Without it, those exceptions would just flow through.